src/pages/[marketplace]/invest-withdraw.tsx

```diff
@@ -36,7 +36,9 @@ function InvestWithdraw() {
 InvestWithdraw.getLayout = function getLayout(page: ReactElement) {
 	return (
 		<Layout.WithSideMenu>
-			<Layout.WithMainMenu showLogo={false}>{page}</Layout.WithMainMenu>
+			<Layout.WithMainMenu showLogo={false}>
+				<Layout.WithContainer>{page}</Layout.WithContainer>
```

Comment:
I would say that the container stuff should always be present when there is content.
and whenever there is content we are at least using the mainmenu layout. I would just put the padding stuff around the children of the mainmenu layout instead of adding a separate layout.

Reply:
yeah the thing is we sometimes have special cases that don't require the container. See the market index page for example. That layout has a lot less padding on top and is wider. 
So I saw three options: 
1. Don't use the composable layout system on those pages and just add the mainmenu layout in the page component
2. Add the container code to the mainmenu and change the layout of the market index page
3. Add a withContainer layout so we can maintain composability

I don't have a strong preference for either solution so if you want I can change it for option 1. (or if you have another suggestion feel free to tell me)

Reply:
I prefer number 2 but let's run it by Tim first